src/librustdoc/html/static/main.js

```diff
@@ -1513,4 +1515,12 @@ function copy_path(but) {
     document.body.removeChild(el);
 
     but.textContent = '✓';
+    
+    window.clearTimeout(reset_button_timeout);
```

Comment:
This won't error the first time: <https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout#notes>.